apis/python/src/tiledb/vector_search/ivf_flat_index.py

```diff
@@ -466,7 +466,6 @@ def create(
         group_exists=group_exists,
         config=config,
     )
-    # TODO(paris): Save training_source_uri as metadata so that we use it for re-ingestion's.
```

Comment:
We ended up going with a different design (i.e. `copy_centroids_uri`), so do not need this.